src/Explorer/Explorer.tsx

```diff
@@ -348,7 +348,7 @@ export default class Explorer {
     this._isInitializingNotebooks = true;
     if (userContext.features.phoenix) {
       const connectionStatus: DataModels.ContainerConnectionInfo = {
```

Comment:
do we need this? Lets remove it and control it fully in phoenix client, we dont need a connecting state at all, then for the user whats the diff between connecting and allocating, why is the timer running for only one as opposed to the other?

So lets remove connecting state and the spinner

Reply:
ok removed.